database/database.go

```diff
@@ -0,0 +1,90 @@
+package database
+
+import (
+	"bms/model"
+
+	"github.com/jinzhu/gorm"
+)
+
+type DataBase interface {
+	CreateUser(model.User)
+	UpdateUser(model.User)
+	GetShow(int) []model.Show
+	UpdateShow(model.Show)
+	AddSeat(model.Seat)
+	AddPayment(model.Payment) model.User
+	AddMovie(model.Movie)
+	GetMovies() []model.Movie
+	GetMovie(model.MoviePreference) []model.Movie
+	UpdateMovie(model.Movie)
+	AddBooking(model.Booking)
+	GetBookings(int) []model.Booking
+	CancelBooking(int)
+}
+
+type DBClient struct {
+	Db *gorm.DB
+}
+
+func (db DBClient) CreateUser(user model.User) {
+	db.Db.Save(&user)
```

Comment:
Add return type status,error for all such functions. If db call is successful return status,nil. Else return error

Reply:
created .env and stored critical data into it

Reply:
added return types to the functions for error handling